### PR TITLE
Improved Explanation of Labels and Annotations

### PR DESCRIPTION
**Pull request by khubaibalam2000**
This Pull Request resolved the issue mentioned in #42524. I have added explanation and it makes clear that both labels and annotations can be used together.

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
@khubaibalam2000 Thank you for your contribution!
It appears that this PR includes changes to files across multiple languages _('en', 'zh')_. However, docs follows different processes for each localization, and and we typically don't accept pull requests that impact multiple languages simultaneously. I would recommend modify only the primary website content (which means only English content) initially with this PR. After the merge of the English content, we could create separate pull request for the remaining languages. I hope this suggestion is helpful.

**Comment by khubaibalam2000** [comment]
Thank You @k8s-ci-robot for your guidance. I have consulted with the PR process documentation and also signed a **CLA** document.

**Review by aj11anuj — COMMENTED** [review]
You have made changes to ("en" and "zh-cn") 2 files. Make another commit in which you have to revert changes of "zh-cn" localized file. So, only "en" file remain changed.

Screenshot (794)

**Comment by khubaibalam2000** [comment]
@aj11anuj Thank you for pointing out this. In the new commit, I have revert it back to the original one.

File: content/en/docs/concepts/overview/working-with-objects/annotations.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ objects. Labels can be used to select objects and to find
 collections of objects that satisfy certain conditions. In contrast, annotations
 are not used to identify and select objects. The metadata
 in an annotation can be small or large, structured or unstructured, and can
-include characters not permitted by labels.
+include characters not permitted by labels. It is also possible to use labels with 
+annotations to attach metadata to Kubernetes objects
```
Comment (tengqm) [inline content/en/docs/concepts/overview/working-with-objects/annotations.md:21]: 1. Missing period for the sentence.
2. What do you mean by "use labels with annotations"?

Comment (khubaibalam2000) [reply content/en/docs/concepts/overview/working-with-objects/annotations.md]: @tengqm Thank you for pointing out the mistake. I have added missing period for the sentence. Moreover, I have made the sentence more clearer in the latest commit. As mentioned in the issue #42524, we can use labels as well as annotations in metadata of same object. Therefore, I added a clearer sentence in the end of the paragraph.

File: content/en/docs/concepts/overview/working-with-objects/annotations.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ objects. Labels can be used to select objects and to find
 collections of objects that satisfy certain conditions. In contrast, annotations
 are not used to identify and select objects. The metadata
 in an annotation can be small or large, structured or unstructured, and can
-include characters not permitted by labels.
+include characters not permitted by labels. It is also possible to use labels as 
+well as annotations in metadata of same object.
```
Comment (tengqm) [inline content/en/docs/concepts/overview/working-with-objects/annotations.md:21]:
```suggestion
well as annotations in the metadata of the same object.
```

File: content/en/docs/concepts/overview/working-with-objects/annotations.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ objects. Labels can be used to select objects and to find
 collections of objects that satisfy certain conditions. In contrast, annotations
 are not used to identify and select objects. The metadata
 in an annotation can be small or large, structured or unstructured, and can
-include characters not permitted by labels.
+include characters not permitted by labels. It is also possible to use labels as
```
Comment (tengqm) [inline content/en/docs/concepts/overview/working-with-objects/annotations.md:20]:
```suggestion
include characters not permitted by labels. It is possible to use labels as 
```

**Comment by khubaibalam2000** [comment]
@tengqm Thank You for your guidance. In the latest commit, I fixed those mistakes. You can review now.

**Pull request merged**